src/jshint.js

```diff
@@ -1340,8 +1340,10 @@ var JSHINT = (function() {
       while (!obj.identifier && typeof obj.left === "object")
         obj = obj.left;
 
-      if (obj.identifier && natives.indexOf(obj.value) >= 0)
+      if (obj.identifier && natives.indexOf(obj.value) >= 0 &&
+          state.funct["(scope)"].isPredefined(obj.value)) {
```

Comment:
You make a good point. The semantics of this function necessarily involve state outside of the expression itself!